DESCRIPTION

```diff
@@ -1,16 +1,21 @@
-Package: rsconnect
 Type: Package
-Title: Deployment Interface for R Markdown Documents and Shiny Applications
+Package: rsconnect
+Title: Deployment Interface for R Markdown Documents and Shiny
```

Comment:
Maybe the title should be reworded? CC @kelli-rstudio 

> Deployment support for RPubs, Posit Connect, and shinyapps.io

Reply:
Sounds good to me. Feel free to merge and we can adjust if @kellobri has other feedback.